internal/commands/permission.go

```diff
@@ -76,29 +76,34 @@ func RegisterPermissionCmd(rootCmd *cobra.Command) *cobra.Command {
 	checkCmd.Flags().Bool("schema", false, "requests debug information from SpiceDB and prints out the schema used")
 	checkCmd.Flags().Bool("error-on-no-permission", false, "if true, zed will return exit code 1 if subject does not have unconditional permission")
 	checkCmd.Flags().String("caveat-context", "", "the caveat context to send along with the check, in JSON form")
+	checkCmd.Flags().String("request-id", "", "optional id to send along with the request for tracing")
```

Comment:
Can't we turn this into a global flag so it does not have to be repeated for each subcommand?